src/sounds.cpp

```diff
@@ -283,11 +283,8 @@ void sounds::process_sound_markers( player *p )
         }
         // Only print a description if it exists
         if( !description.empty() ) {
-            // If it came from us, don't print a direction
             if( pos == p->pos() ) {
-                std::string uppercased = description;
-                capitalize_letter( uppercased, 0 );
-                add_msg( "%s", uppercased.c_str() );
+                add_msg( "You hear %s", description.c_str());
```

Comment:
Sorry, when I said "below you" in the title, I actually meant "at your current position". I'll update it to reflect that.  
You're right, I completely forgot about the custom sound message prefixes. 
At this point, it would be better to close this PR and open another which would address the matter completely.